pkg/storage/gcp/README.md

```diff
@@ -0,0 +1,28 @@
+# Google Cloud Storage Driver
+
+This driver provides support for storing module files in Google Cloud storage.
+You may host as little as the storage only on GCP, the entire project does not need to run there.
```

Comment:
@robjloranger I don't quite understand this sentence. Are you saying that you can store some other stuff outside of GCP?

Reply:
I should reword it. I mean to convey that you can run olympus on say a server and use a cloud storage bucket for the modules.